hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -2139,6 +2139,15 @@ private LongStream getStoreFileAgeStream() {
         .map(t -> EnvironmentEdgeManager.currentTime() - t);
   }
 
+  @Override
+  public int getStoreRefCount() {
+    return this.storeEngine.getStoreFileManager().getStorefiles().stream()
+      .filter(sf -> { return sf.getReader() != null; })
```

Comment:
`sf -> sf.getReader() != null` is enough?

Reply:
Ok. This style is new to me